src/components/Application-Header/JobsAppBar.tsx

```diff
@@ -1,30 +1,94 @@
 import * as React from 'react';
-
+import { withStyles, WithStyles, createStyles } from '@material-ui/core';
 import AppBar from '@material-ui/core/AppBar';
 import Toolbar from '@material-ui/core/Toolbar';
 import Typography from '@material-ui/core/Typography';
-
+import AccessibilityNewIcon from '@material-ui/icons/AccessibilityNew';
+import Avatar from '@material-ui/core/Avatar';
+import IconButton from '@material-ui/core/IconButton';
+import StarIcon from '@material-ui/icons/Star';
+import Tooltip from '@material-ui/core/Tooltip';
 import { User } from '../../types/userTypes';
 
-interface AppBarData {
+const styles = () => createStyles({
+    root: {
+        backgroundColor: 'gray'
+    },
+    toolbar: { 
+        display: 'flex',
+        justifyContent: 'space-between'
+    },
+    avatar: {
+        backgroundColor: 'rgb(87,87,87)',
+        margin: '0 5px'
+    },
+    starIconYellow:{
+        color: 'yellow'
+    },
+    starIconWhite:{
+        color: 'white'
+    },
+    userDetails:{
+        display: 'flex',
+        alignItems: 'center'
+    },
+    logoContainer: {
+        display: 'flex',
+        alignItems: 'center'
+    },
+    logoIcon:{
+        margin: '0 5px'
+    }
+});
+
+interface AppBarData extends WithStyles<typeof styles> {
     user: User;
 }
 
-type AppBarProps = AppBarData;
+type AppBarProps = AppBarData //&& withStyles<typeof styles>
 
-export const JobsAppBar: React.FC<AppBarProps> = (props): JSX.Element => {
-    const { user } = props;
+const JobsAppBar: React.FC<AppBarProps> = (props): JSX.Element => {
+    
+    // For now I used a stupid function, just to get a different color   
+    const userHaveFavorites = (): boolean =>{
+        return Math.random() >= 0.5;
+    }
+    
+    const { user, classes } = props;
 
     return (
-        <AppBar position="static">
-            <Toolbar>
-                <Typography variant="h5">
-                    {user.userInitials}
-                </Typography>
-                <Typography variant="h3">
-                    Meshushim
-                </Typography>
+        <AppBar className={classes.root} position="static">
+            <Toolbar className={classes.toolbar}>
+                <div className={classes.userDetails}>
```

Comment:
Split to method getUserDetails? 😄

Reply:
Done in  b2dae38abe8cdeb64cc1b3a8fc7f1c469cfdad29

Reply:
Thanks good idea